tests/functional/tests/mgr/test_mgr.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+
+
+class TestMGRs(object):
+
+    @pytest.mark.no_docker
+    def test_mgr_is_installed(self, node, host):
+        assert host.package("ceph-mgr").is_installed
+
+    def test_mgr_service_is_running(self, node, host):
+        service_name = "ceph-mgr@{hostname}".format(
+            hostname=node["vars"]["inventory_hostname"]
+        )
+        assert host.service(service_name).is_running
+
+    def test_mgr_service_is_enabled(self, node, host):
+        service_name = "ceph-mgr@{hostname}".format(
+            hostname=node["vars"]["inventory_hostname"]
+        )
+        assert host.service(service_name).is_enabled
+
+    @pytest.mark.no_docker
+    def test_mgr_is_up(self, node, host):
+        cmd = "sudo ceph --name mgr.{hostname} --keyring /etc/ceph/{cluster}.mgr.{hostname}.keyring --cluster={cluster} --connect-timeout 5 -f json -s | python -c 'import sys, json; print(json.load(sys.stdin)[\"mgrmap\"][\"active_name\"])'".format(
+            hostname=node["vars"]["inventory_hostname"],
+            cluster=node["cluster_name"]
+        )
+        output = host.check_output(cmd)
+        phrase = "{hostname}".format(hostname=node["vars"]["inventory_hostname"])
+        assert phrase in output
+
+    @pytest.mark.docker
+    def test_mgr_is_up(self, node, host):
```

Comment:
`test_mgr_is_up()` is redefined here, it means it will never be tested on non-containerized deployment.